modules/ROOT/pages/data-bridge.adoc

```diff
@@ -8,133 +8,122 @@ The data bridge is the secure connection from Anypoint API Community Manager to
-----
-+
-. The `cURL` command returns a response like this:
-+
-----
-    {
-      "client_id": "5fafXXXXXXXXXXXXXXXXXXXXXXXX29c9",
-      "client_secret": "9509XXXXXXXXXXXXXXXXXXXXXXXXC10E",
-      "client_name": "ACM Integration App",
-      "redirect_uris": [
-        "https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"
-      ],
-      "grant_types": [
-        "authorization_code",
-        "refresh_token"
-      ],
-      "public_keys": [],
-      "scopes": [
-        "offline_access",
-        "full"
-      ],
-      "enabled": true,
-      "owner_org_id": "f377XXXX-XXXX-XXXX-XXXX-XXXXXXXX9d08",
-      "as_id": "anypoint"
-    }
-----
-+
-. Note the `client_id` and `client_secret` values.
+If you already have Anypoint Identity Provider, get identity provider data from your current configuration:
 
-== Create an Authentication Provider in Your Salesforce Organization [[create-auth-provider]]
+. Click *Access Management* > *Identity Providers*.
+. On *SAML 2.0* click *Edit*.
+. Copy the issuer and the audience so you can use them in the following steps.
 
-In the following task steps, if your Anypoint Platform organization is hosted in the EU, replace all instances of `+https://anypoint.mulesoft.com+` with `+https://eu1.anypoint.mulesoft.com+`.
+== Configure Named Credential
 
-. In *Setup*, search for *Auth. Providers* using the *Quick Find* box.
-. Click *Auth. Providers* -> *New*.
-. In *Provider type*, select *Open ID Connect*.
-. Set *Name* to `Anypoint`.
-. Set *Consumer Key* to the application ID of the connected app (which you took note of when you created the app in Anypoint Platform).
-+
-This ID was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
-+
-. Set *Consumer Secret* to the application secret of the connected app created in Anypoint Platform.
+. Click *Setup* > *Named Credentials*.
+. Modify the named credential *anypoint_mulesoft*.
+. Set *URL* to `+https://anypoint.mulesoft.com+`.
+. Go to the *Authentication* section.
+. Set *Identity Type* to *Per User*.
+. Set *Authentication Protocol* to *JWT Token Exchange*.
+. Set *Token Endpoint Url* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
+. Set *Issuer* to the client ID of the connected app you saved previously.
+. Set *Per User Subject* to the following formula.
 +
-This secret was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
+Replace `IDP_ISSUER` with the issuer from your IdP configuration. Replace `IDP_AUDIENCE` with the audience from your IdP configuration.
```

Comment:
". Set *Per User Subject* to the following, replacing `IDP_ISSUER` with the issuer and `IDP_AUDIENCE` with the audience from your IdP configuration: